themes/hugo-planetarium/layouts/_default/single.html

```diff
@@ -33,11 +33,14 @@ <h1 itemprop="name" id="title" class="mb1">{{ .Title }}</h1>
         )
       {{ end }}
     </div>
-
     <article itemprop="articleBody" id="content" class="w-100 lh-copy">
       {{ .Content }}
     </article>
-
+    {{ if (eq .Language.Lang "kor") }}
+    <div class="recruit">
+      <a href="https://planetariumhq.com">플라네타리움</a>은 게임에 특화된 오픈 소스 P2P 라이브러리 <a href="https://libplanet.io/">Libplanet</a>과, 그 위에서 중앙 서버 없는 온라인 게임 <a href="http://nine-chronicles.com/">Nine Chronicles</a>를 만들고 있습니다. 저희와 흥미로운 기술적 도전을 함께 하실 분들을 모시고 있습니다. 지금 <a href="http://bit.ly/join-planet">채용 페이지</a>를 확인해주세요!
```

Comment:
사소하지만 저희 스낵이 https를 쓰고 있어서, 아마 https → http로 링크하면 방문 추적이 안 될 겁니다. 그래서 https 주소로 링크하는 게 좋을 것 같습니다.

```suggestion
      <a href="https://planetariumhq.com">플라네타리움</a>은 게임에 특화된 오픈 소스 P2P 라이브러리 <a href="https://libplanet.io/">Libplanet</a>과, 그 위에서 중앙 서버 없는 온라인 게임 <a href="http://nine-chronicles.com/">Nine Chronicles</a>를 만들고 있습니다. 저희와 흥미로운 기술적 도전을 함께 하실 분들을 모시고 있습니다. 지금 <a href="https://bit.ly/join-planet">채용 페이지</a>를 확인해주세요!
```